### PR TITLE
:hammer: [#1215] Add script to dump config from database

### LINKED ISSUE #1215: As admin, I want to be able to easily export Open Zaak configuration (from the admin UI)
... so I can keep the configuration in sync between test/staging/prod environments

**Description**
`./bin/dump_configuration.sh` should dump a fixture containing all the configuration that can be done in the admin:

* zgw_consumers.service
* zgw_consumers.certificate
* zgw_consumers.nlxconfig
* drc_cmis.cmis_config
* feature_flags
* selectielijstconfig
* openid connect config

### PR DESCRIPTION
**Pull request by stevenbal**
Requires https://github.com/maykinmedia/zgw-consumers/pull/44 to be merged and zgw_consumers version to be bumped

Fixes #1215

**Changes**

* Add script to dump config from database

### CODE REVIEW

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I think we're missing:

* `vng_api_common.JWTSecret`
* `autorisaties.Applicatie` & related permissions (so that other services can connect to Open Zaak)
* `notifications.NotificatiesConfig` & webhook subscriber information (latter should be empty though?)

File: bin/dump_configuration.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+python src/manage.py dumpdata --indent 4 \
+                zgw_consumers.Service \
+                zgw_consumers.Certificate \
```
[the remark this answers is not in the document]
Comment (stevenbal) [reply bin/dump_configuration.sh:5]: Would dumping the files to a .zip archive work? https://github.com/maykinmedia/zgw-consumers/pull/44

[the remark this answers is not in the document]
Comment (sergei-maertens) [reply bin/dump_configuration.sh:5]: yeah, should be fine - as long as the files can be matched & imported again (relatively) easily

**Comment by sergei-maertens** [comment]
zgw-consumers 0.19.0 was published today!

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Can you make the script a bit more in line with the version in Open Forms: https://github.com/open-formulieren/open-forms/blob/master/bin/dump_configuration.sh

* Creating a temporary directory to dump to
* Support `TMPDIR` environment variable

-> this makes it so we can mount a volume and don't need to write to the container file system, which is usually advised against.

* Include comments in the script on how to use it, and ensure the certificate files are dumped as well.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**